index.js

```diff
@@ -0,0 +1,177 @@
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+
+const loadTrips = () => {
+  reportStatus('Loading trips...');
+
+  const allTrips = $('#every-trip');
+  allTrips.empty();
+
+  let allTripsUrl = BASE_URL
+
+  axios.get(allTripsUrl)
+    .then((response) => {
+      reportStatus(`Successfully loaded ${response.data.length} trips`);
+      response.data.forEach((trip) => {
+        allTrips.append(`<li id="${trip.id}">${trip.name}</li>`);
+      });
+    })
+    .catch((error) => {
+      reportStatus(`Encountered an error while loading trips: ${error.message}`);
+      console.log(error);
+    });
+};
+
+const getTripDetail = function getTripDetail(tripID) {
+
+  reportStatus('Getting trip details.');
+
+  targetTripId = tripID
+
+  const singleTripDetail = $('#trip-deets');
+  singleTripDetail.empty();
+
+  let tripDetailUrl = BASE_URL.concat("/").concat(tripID)
+
+  const reserveButton = $('#reserve-button');
+  reserveButton.empty();
+
+  console.log(`${tripDetailUrl}`);
+
+  axios.get(tripDetailUrl)
+    .then((response) => {
+      reportStatus(`Trip details loaded!`);
+      targetTripName = response.data.name;
+      singleTripDetail.append(
+        `<h1>HERE'S ONE GOOD WAY TO GET TREKKED UP:</h1>
+        <section class="details_box" >
+          <h3>Name: ${targetTripName}</h3>
+          <h4>Continent: ${response.data.continent}</h4>
+          <h4>Category:  ${response.data.category}</h4>
+          <h5>Weeks: ${response.data.weeks}</h5>
+          <h5>Cost: ${response.data.cost}</h5>
+          <h5>About: </h5>
+          <p>${response.data.about}</p>
+          <h6>Trip ID: ${targetTripId}</h6>
+        </section>`);
+    })
+    .catch((error) => {
+      reportStatus(`Encountered an error while loading trip details: ${error.message}`);
+      console.log(error);
+    });
+    if (singleTripDetail.length > 0 ) {
+      reserveButton.append(`<button id="reservation">I Would Like To Trek Up in This Manner</button>` )
+    }
+};
+
+
+const callReservationScreen = function callReservationScreen() {
+
+    reportStatus('Pulling up reservation screen');
+
+    const reservationScreen = $('#reservation-box');
+    reservationScreen.empty();
+
+    reservationScreen.append(
+      ` <h1> SECURE YOUR OPPORTUNITY TO TREK UP ROYALLY: </h1>
+        <h3 class="details_box"> The Trip You Want To Go On Is: ${targetTripName}</h3>
+        <form class="details_box" id='reservation-form' name='reservation-form'>
+          <label for='name'>Name:</label>
+          <input type='text' id='name' name='name'><br>
+          <label for='email'>Email:</label>
+          <input type='email' id='email' name='email'><br>
+          <label>&nbsp;</label>
+          <input type='submit' id='submit-reservation-info'
+          value='I hereby commit to unapologetically trekking up my life'><br>
```

Comment:
You use `<br>` tags here... which isn't a no-no, it's quite common to use those in forms, but have you considered using ~css~ to make them all `display: block;`?